src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is started!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck is stopped!");
```

Comment:
Similarly, the message 'Truck is stopped!' could be more informative. Consider specifying what has stopped, such as 'Truck's engine is stopped' to clarify the action.